tests/spine_db_editor/test_shortcuts.py

```diff
@@ -0,0 +1,26 @@
+import unittest
```

Comment:
Nice to have tests too! Maybe add them to `test_SpineDBEditor.py` instead? Besides, this file is missing copyright notice.